packages/fluent-editor/src/modules/ai/index.ts

```
@@ -216,68 +686,122 @@
+      this.actionMenuEl.style.display = 'none'
+    }
+
+    this.showResultPopupEl = false
+
+    if (this.inputEl && this.inputEl.value.trim() !== '') {
+      this.inputEl.value = '' // 清空输入框
+    }
+    this.hideSelectionBubble()
+  }
+
+  set charCount(value: number) {
```

ESLint

[error] Getter is not present for class setter 'charCount'.

(accessor-pairs)